src/App.js

```diff
@@ -6,6 +6,7 @@ import Reviews from "./components/Reviews";
 import Categories from "./components/Categories";
```

Comment:
Check your imports here, you are referencing a folder called 'components' but your folder is actually called 'Components' with a capital C. Casing matters when referencing file names